src/Kconfig

```diff
@@ -48,7 +49,14 @@ config GOLIOTH_OTA_OBSERVATION_RETRY_MAX_DELAY_S
         backoff doubling between retry attempts. The delay value will increase until this limit is
```

Comment:
Can't comment on the exact line, but `GOLIOTH_OTA_OBSERVATION_RETRY_MAX_DELAY_S` is also used by the firmware update module, not OTA.